test/makeRequest-test.js

```diff
@@ -17,6 +17,17 @@ let errorTries;
 
 beforeAll(async () => {
   httpServer = http.createServer((req, res) => {
+    if (req.url === '/timeout') {
+      setTimeout(() => {
+        res.writeHead(200, { 'Content-Type': 'application/json' });
+        res.end(
+          JSON.stringify({
+            result: 'Hello world!',
+          }),
+        );
+      }, 1000);
```

Comment:
This matches the timeout value so it could lead to a flaky test, right? Maybe increase this a bit?

Reply:
It's 999ms longer, so should be fine. 🙂

Reply:
Ah! For some reason I thought the timeout was in seconds and this was ms. That's what I get for reviewing code right when I wake up. lol